share/fathead/git_manual/output.txt

```diff
@@ -0,0 +1,413 @@
+git-config	A										Get and set repository or global options
+<pre><code>git config [<file-option>] [type] [-z|--null] name [value [value_regex]]
+git config [<file-option>] [type] --add name value
+git config [<file-option>] [type] --replace-all name value [value_regex]
+git config [<file-option>] [type] [-z|--null] --get name [value_regex]
+git config [<file-option>] [type] [-z|--null] --get-all name [value_regex]
+git config [<file-option>] [type] [-z|--null] [--name-only] --get-regexp name_regex [value_regex]
+git config [<file-option>] [type] [-z|--null] --get-urlmatch name URL
+git config [<file-option>] --unset name [value_regex]
+git config [<file-option>] --unset-all name [value_regex]
+git config [<file-option>] --rename-section old_name new_name
+git config [<file-option>] --remove-section name
+git config [<file-option>] [-z|--null] [--name-only] -l | --list
+git config [<file-option>] --get-color name [default]
+git config [<file-option>] --get-colorbool name [stdout-is-tty]
+git config [<file-option>] -e | --edit</pre></code>	https://git-scm.com/docs//config.html
```

Comment:
Actually can you make that a literal `\n` to represent newlines.  I fixed the first one manually and put it on beta.  Here's what the entry in output.txt should look like. 
```
git config      A                                                                               Get and set repository or global options <pre><code>git config [<file-option>] [type] [-z|--null] name [value [value_regex]]\ngit config [<file-option>] [type] --add name value\ngit config [<file-option>] [type] --replace-all name value [value_regex]\ngit config [<file-option>] [type] [-z|--null] --get name [value_regex]\ngit config [<file-option>] [type] [-z|--null] --get-all name [value_regex]\ngit config [<file-option>] [type] [-z|--null] [--name-only] --get-regexp name_regex [value_regex]\ngit config [<file-option>] [type] [-z|--null] --get-urlmatch name URL\ngit config [<file-option>] --unset name [value_regex]\ngit config [<file-option>] --unset-all name [value_regex]\ngit config [<file-option>] --rename-section old_name new_name\ngit config [<file-option>] --remove-section name\ngit config [<file-option>] [-z|--null] [--name-only] -l | --list\ngit config [<file-option>] --get-color name [default]\ngit config [<file-option>] --get-colorbool name [stdout-is-tty]\ngit config [<file-option>] -e | --edit</pre></code>  https://git-scm.com/docs//config.html
```
https://beta.duckduckgo.com/?q=git+config&t=canonical&ia=about
![selection_300](https://cloud.githubusercontent.com/assets/1882892/13986638/e7533cfe-f0d9-11e5-8edf-5846952e917f.jpg)